apps/rehabstod/src/pages/Patient/components/patientOverview/PatientOverview.tsx

```diff
@@ -24,12 +54,27 @@ export function PatientOverview({ sjfMetaData, patientId }: { sjfMetaData: SjfMe
         items={sjfMetaData.kraverInteSamtycke}
         onGetInformation={handleGetCareUnitInformation}
       />
+      <OpenInformationWithConsentCard
+        items={sjfMetaData.kraverSamtycke}
+        title="Ospärrad information hos annan vårdgivare"
+        subTitle="Vårdgivare med information"
+        description="Det finns ospärrad information hos en annan vårdgivare. Du kan klicka nedan för att visa vilka vårdgivare som har ospärrad information. Patientens samtycke krävs för att du ska kunna ta del av informationen."
+        hasGivenConsent={sjfMetaData.samtyckeFinns || responseCode === 'OK'}
```

Comment:
Instead of having this state in two request, could we invalidate `SickLeavePatient` or use `onQueryStarted` to manipulate `getSickLeavePatient` similar to `updateUserPreferences`?